SpatialGDK/Source/Public/Interop/SpatialReceiver.h

```diff
@@ -135,7 +135,7 @@ class USpatialReceiver : public UObject
 
 	void ReceiveActor(Worker_EntityId EntityId);
 	void RemoveActor(Worker_EntityId EntityId);
-	AActor* CreateActor(improbable::Position* Position, struct improbable::Rotation* Rotation, UClass* ActorClass, bool bDeferred);
+	AActor* CreateActor(improbable::Position* Position, struct improbable::SpawnData* SpawnData, UClass* ActorClass, bool bDeferred);
```

Comment:
Why are we forward declaring the SpawnData but not the position?

Reply:
Probably because it was already declared. We could forward declare the position as well.